src/app/(nextjs_migration)/components/client/FalseDoorTest.module.scss

```diff
@@ -0,0 +1,80 @@
+@import "../../../tokens.scss";
+
+.falseDoorTestWrapper {
+  --monitorLogoSizeSmall: 30px;
+  --monitorLogoSizeBig: 50px;
+  font: $text-body-md;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  z-index: 2;
+  padding-inline: $spacing-lg;
+  padding-block: $spacing-md;
+  width: 100%;
+  align-items: center;
+  background-color: #ffdff0;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  gap: $spacing-xl;
+
+  @media screen and (min-width: $screen-lg) {
+    padding-inline: $spacing-2xl;
+    padding-block: $spacing-md;
+  }
+
+  .dismiss {
+    align-self: flex-start;
+    margin-left: auto;
+    color: $color-grey-40;
+  }
+
+  .content {
+    display: flex;
+    flex-direction: row;
+    flex: column;
+    flex-wrap: wrap;
+    align-items: center;
+    gap: $spacing-md;
+    width: 100%;
+    justify-content: space-between;
+
+    .imageAndCopy {
+      display: flex;
+      gap: $spacing-md;
+    }
+
+    @media screen and (min-width: $screen-lg) {
+      padding-left: $spacing-2xl;
+    }
+
+    .logo {
+      width: var(--monitorLogoSizeSmall);
+      height: var(--monitorLogoSizeSmall);
+
+      @media screen and (min-width: $screen-lg) {
+        width: var(--monitorLogoSizeBig);
+        height: var(--monitorLogoSizeBig);
+      }
+    }
+
+    p {
+      margin: 0;
+    }
+
+    .cta {
+      all: unset;
```

Comment:
Question: This is hiding visual focus from the user — can we add another focus state if the default one does not have the desired behavior?